test/kitchen/tasks/run-test-kitchen.sh

```diff
@@ -46,25 +46,25 @@ if [ "$KITCHEN_PROVIDER" == "azure" ]; then
   # These should not be printed out
   set +x
   if [ -z ${AZURE_CLIENT_ID+x} ]; then
-    AZURE_CLIENT_ID=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.azure_kitchen_client_id --with-decryption --query "Parameter.Value" --out text)
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏

Same here, to confirm that this is not used locally, otherwise it will not work anymore if $CI_PROJECT_DIR is not set when running locally

Reply:
Here as well, I'll update it, thanks